jck/jtrunner/JavaTestRunner.java

```diff
@@ -1220,7 +1243,7 @@ private static boolean testsRequireDisplay (String tests) {
 	private static String getTestSpecificJvmOptions(String jckVersion, String tests) {
 		String testSpecificJvmOptions = "";
 		
-		if ( tests.contains("api/javax_net") || tests.contains("api/javax_xml") || (getJckVersionInt(jckVersionNo) >= 18 && (tests.contains("api/java_net") || tests.contains("api/java_util"))) ) {
+		if ( tests.contains("api/javax_net") || tests.contains("api/javax_xml") || tests.contains("api/java_net") || tests.contains("api/java_util") ) {
```

Comment:
By removing `getJckVersionInt(jckVersionNo) >= 18`, we will set extra security.properties for all version. I thought we only want to set for JDK11+.

Reply:
Thanks @llxia ! Yes, I've added the version check back, and changed the value to 11+.